tests/e2e/manifests/verify_manifest_lists.go

```diff
@@ -631,6 +631,7 @@ func filterVersions(versions VersionList) (VersionList, error) {
 		if err != nil {
 			return versions, err
 		}
+		verFromURL = verFromURL[1:len(verFromURL)-1]
```

Comment:
> "panic: could not parse "v1.18.6\n" as version [recovered]"

where are you seeing this panic?

better to use:
```
verFromURL = strings.TrimSpace(verFromURL)
```

the leading "v" would be parsed.

Reply:
I find this panic when doing #2243 
Thanks and done